config/config.go

```diff
@@ -90,6 +90,8 @@ type BaseConfig struct {
 
 	SyncRequestTimeout int `mapstructure:"sync-request-timeout"` // the timeout for direct request in the sync
 
+	SyncInterval int `mapstructure:"sync-interval"`
```

Comment:
Specify the units of this interval (seconds probably).

Reply:
cool ill add some documentation